src/browser/components/MattermostView.jsx

```diff
@@ -126,8 +126,14 @@ export default class MattermostView extends React.Component {
           shell.openExternal(e.url);
         }
       } else {
-        // if the link is external, use default os' application.
-        ipcRenderer.send('confirm-protocol', destURL.protocol, e.url);
+        const parsedURL = Utils.parseURL(e.url);
+        const serverURL = Utils.getServer(parsedURL, this.props.teams);
+        if (serverURL !== null && Utils.isTeamUrl(serverURL.url, parsedURL)) {
```

Comment:
Should `serverURL !== null` be `serverURL != null` so that it handles undefined as well?

Reply:
`getServer()` explicitly returns null, so I think we're okay here.